stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,990 @@
+||<code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_summarized-title-area--no-shadow</code>        |
+| Add shadow to tabs when tabs are used | <code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_tabs--add-shadow</code>       |
+| Collapsed styles when header is collapsed | <code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_title-area--collapsed</code>      |
+||<code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_title--collapsed</code>        |
+||<code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_summarized-title--collapsed</code>        |
+| The pin/collapse button group | <code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_header-visibility-container--button-group</code>      |
+| The left gradient of the button group | <code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_header-visibility-container--left-gradient</code>      |
+| The right gradient of the button group | <code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_header-visibility-container--right-gradient</code>      |
+## Background variants
+
+| **Description** | <span style="margin-left: 2rem;">**Modifier class**</span> |
+| :---- | :-------------- |
+| List background for content area|<code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_content--list-bg</code>        |
+| Transparent background for title area | <code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_title-area--transparent-bg</code> |
+| Transparent background for header area|<code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_header--transparent-bg</code>        |
+| Transparent background for content area|<code class="docs-code" style="margin-left: 2rem;">fd-dynamic-page\\_\\_content--transparent-bg</code>        |
+
+`,
+        tags: ['f3', 'theme', 'development']
+    }
+};
+
+export const desktop = () => `
+    <div class="fd-dynamic-page" role="region" aria-label="Dynamic Page Desktop Example">
+        <div class="fd-dynamic-page__header">
+            <div class="fd-dynamic-page__title-area fd-dynamic-page__title-area--xl" tabindex="0">
+                <ul class="fd-dynamic-page__breadcrumb fd-breadcrumb">
+                    <li class="fd-breadcrumb__item"><a class="fd-breadcrumb__link" tabindex="0" href="#">Men</a></li>
+                    <li class="fd-breadcrumb__item"><a class="fd-breadcrumb__link" tabindex="0" href="#">Shoes</a></li>
+                    <li class="fd-breadcrumb__item"><a class="fd-breadcrumb__link" tabindex="0" href="#">Running Shoes</a></li>
+                </ul>
+                <div class="fd-dynamic-page__title-container">
+                    <div class="fd-dynamic-page__title"> Balenciaga Tripple S Trainers </div>
```

Comment:
The problem with this is `h1` gives its font-weight: bold attribute to the title since our `fd-dynamic-page__title` class does not override this, making it look off. Also, the title can also constitute a link at times(as seen in tablet example); therefore this cannot be applied without making additional changes to styles that are not specified in the spec.

Reply:
Is it not possible to use the [Title component](https://deploy-preview-1598--fundamental-styles.netlify.app/?path=/docs/components-title--levels) for styling?

Also:
```html
<h1><a href="#">Header Text</a></h1>
```
is semantically valid. We only need to style it according to current spec.

Reply:
Was not aware of using Title component. Thanks, have updated now. Please check.